Desktop/engine/enginerepresentation.cpp

```diff
@@ -460,6 +460,10 @@ void EngineRepresentation::runAnalysisOnProcess(Analysis *analysis)
 
 void EngineRepresentation::analysisRemoved(Analysis * analysis)
 {
+	Log::log() << "Analysis removed" << " it was " << (_analysisAborted == analysis ? "" : " not " ) << "_analysisAborted" << std::endl;
```

Comment:
```suggestion
//	Log::log() << "Analysis removed" << " it was " << (_analysisAborted == analysis ? "" : " not " ) << "_analysisAborted" << std::endl;
```

I guess this comment isnt needed so badly